README.md

```diff
@@ -3,6 +3,71 @@
   <img alt="Open Sauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
   <h1>🍕 Open Sauced Browser Extension(s) 🍕</h1>
```

Comment:
@Anush008 We don't need the (s) anymore ig, now that we know there'll be one extension.

Reply:
I'll update it once the repo name is changed.

Reply:
@bdougie what do you think?

Reply:
Like @0-vortex said, we can rename the repo once we have name for the project.